src/main/java/apitests/steps/common/CommonSteps.java

```diff
@@ -0,0 +1,18 @@
+package apitests.steps.common;
+
+import apitests.clients.PetClient;
+import apitests.model.ResponseDetails;
+import apitests.steps.BaseSteps;
+import io.cucumber.java.en.Then;
+
+public class CommonSteps extends BaseSteps {
+
+    public CommonSteps(ResponseDetails response, PetClient petClient) {
+        super(response, petClient);
+    }
+
+    @Then("The response with status code {int} is returned")
+    public void the_response_with_status_code_is_returned(Integer statusCode) {
+        response.getResponse().then().log().all().statusCode(statusCode);
```

Comment:
each method should be in new line, it will make it easier to read :)

Reply:
IMO this method with one line of code is very simple that we do not need split methods in separate lines.
In clients classes I would like to store more complicated logic for other API calls and there your approach is welcome there :) Or if I would not use the Cucumber approach :) If the team wants your approach we can apply it.

Reply:
I hope it is ok now for you...